runtime/JavaScript/src/antlr4/error/Errors.js

```diff
@@ -78,7 +78,7 @@ LexerNoViableAltException.prototype.constructor = LexerNoViableAltException;
 LexerNoViableAltException.prototype.toString = function() {
     var symbol = "";
     if (this.startIndex >= 0 && this.startIndex < this.input.size) {
-        symbol = this.input.getText((this.startIndex,this.startIndex));
+        symbol = this.input.getText(this.startIndex);
```

Comment:
shouldn't this read 'getText(this.startIndex, this.startIndex)' ?

Reply:
Ah yes. Fixed.